transformations/french_conjugation_transformation/requirements.txt

```diff
@@ -0,0 +1,9 @@
+# for conjugation_transformation
+spacy-lefff==0.4.0
+textblob_fr==0.2.0
+#link to install
+#https://github.com/explosion/spacy-models/releases/download/fr_core_news_lg-3.0.0/fr_core_news_lg-3.0.0-py3-none-any.whl
+fr-core-news-md @ https://github.com/explosion/spacy-models/releases/download/fr_core_news_md-3.0.0/fr_core_news_md-3.0.0-py3-none-any.whl
+mlconjug==3.4.0
+nltk
+scikit-learn==0.21.3
```

Comment:
Hi @Louanes1: Remove the scikit-learn from this file since we already have a latest version included in the main requirements.txt. Try using the global version and test your changes.